Dockerfile.win10.min

```diff
@@ -63,7 +63,6 @@ WORKDIR /
 # option --depth=1 should also be restored
```

Comment:
Need to delete the comment at 61-63 as it is now outdated. Also need to add --depth=1 back to "git clone" command. Is there not a vcpkg release branch that we can use instead of master?

Reply:
Sure we can refer to it as tag for latest release which is in our case will be [vcpkg v2022.03.10](https://github.com/microsoft/vcpkg/tree/2022.03.10)